warehouse/templates/includes/current-user-indicator.html

```diff
@@ -15,7 +15,8 @@
   {% if request.user %}
   <div id="user-indicator" class="horizontal-menu horizontal-menu--light horizontal-menu--tall">
     <nav aria-label="{% trans %}Main navigation{% endtrans %}" class="dropdown dropdown--on-menu dropdown--with-icons">
-      <button type="button" class="horizontal-menu__link dropdown__trigger" aria-haspopup="true" aria-expanded="false" aria-label="{% trans %}View menu{% endtrans %}">
+      <button type="button" class="horizontal-menu__link--with-icon horizontal-menu__link dropdown__trigger" aria-haspopup="true" aria-expanded="false" aria-label="{% trans %}View menu{% endtrans %}">
+        <img class="fa" src="{{ gravatar(request, request.user.email, size=40) }}" alt="User Image">
```

Comment:
I'm not positive on this class name. `fa` is specific to "FontAwesome" our glyphicon set.

I might suggest `class="user-image"` to be nice and explicit.

Reply:
Ah that makes much more sense, I assumed `fa` stood for something else.

Since this is a child of the `horizontal-menu__link--with-icon` class, which could be used for some other thing in the future, would it make more sense to generalize it to either `icn` or `icn-image`? Or does it make more sense to worry about that later and set it to `user-image` for now?

Reply:
I think it's fine to make it `user-image` for now.